bin/shabka

```diff
@@ -0,0 +1,383 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+declare __VERBOSE="${__VERBOSE:-0}"
+
+echoErr() { >&2 echo -e "\e[1;31m[ERR] ${*}\e[0m" ; }
+echoWarn() { >&2 echo -e "\e[1;33m[WARN] ${*}\e[0m" ; }
+echoInfo() { >&2 echo -e "\e[1;34m[INFO] ${*}\e[0m" ; }
+echoDebug1() {
+  if [[ "${__VERBOSE}" -lt 1 ]]; then
+    return
+  fi
+  >&2 echo -e "\e[1;35m[DEBUG1] ${*}\e[0m"
+}
+echoDebug2() {
+  if [[ "${__VERBOSE}" -lt 2 ]]; then
+    return
+  fi
+  >&2 echo -e "\e[1;35m[DEBUG2] ${*}\e[0m"
+}
+echoDebug3() {
+  if [[ "${__VERBOSE}" -lt 3 ]]; then
+    return
+  fi
+  >&2 echo -e "\e[1;35m[DEBUG3] ${*}\e[0m"
+}
+echoSuccess() { >&2 echo -e "\e[1;32m[SUCCESS] ${*}\e[0m" ; }
+containsElement() {
+  declare -r to_find="${1}"; shift 1
+  for element in "${@}"; do [[ "${element}" == "${to_find}" ]] && return 0; done
+  return 1
+}
```

Comment:
I have this function already: https://github.com/kalbasit/shabka/blob/d877e655176c64f2a015a18b8a50543b9c2dd9ae/modules/home/software/zsh/plugins/functions/in_array#L5-L15

Not sure if it's worth refactoring this one. Maybe we can move yours to a helper function along with all the info and debugs

Reply:
This funciton is not even used in the script anymore, removing it.